### PR TITLE
Add beholder

### PR DESCRIPTION
**Pull request by jodusan**
## Linter

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.

### CODE REVIEW

File: python/ray/rllib/models/model.py [hunk 1]
```diff
@@ -65,6 +65,7 @@ def __init__(self,
         self.options = options
         self.scope = tf.get_variable_scope()
         self.session = tf.get_default_session()
+        self.beholder_tensors = []
```
Comment (pimpke) [inline python/ray/rllib/models/model.py]: Create a new class in wgym that derives from model, move this and the method below there, and make all our model extend this new class.

Comment (pimpke) [reply python/ray/rllib/models/model.py]: You can then remove watch_tensor method from wgym/models/common.py

**Review by nikola-j — DISMISSED** [review]
Add readme to wgym

File: python/ray/rllib/evaluation/tf_policy_graph.py [hunk 2]
```diff
@@ -410,8 +410,9 @@ def _build_learn_on_batch(self, builder, postprocessed_batch):
         fetches = builder.add_fetches([
             self._apply_op,
             self._get_grad_and_stats_fetches(),
+            self.model.beholder_tensors
         ])
-        return fetches[1]
+        return fetches[1], fetches[2]
```
Comment (pimpke) [inline python/ray/rllib/evaluation/tf_policy_graph.py]: Override this method in WGymVTracePolicyGraph, and remove these changes from ray.

**Review by pimpke — CHANGES_REQUESTED** [review]

**Review by nikola-j — APPROVED** [review]

**Review by pimpke — APPROVED** [review]

**Pull request merged**